scripts/setup_gce_vm_instance.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+echo ' '
+echo 'This script assists you in creating a Google Cloud Compute Engine VM instance.'
+echo 'Please note that currently this script convenience during DEVELOPMENT by automating'
+echo 'the default options for most GCE VMs and does not include all available options, '
+echo 'such as customizing disk size, type, etc. We could add more features in the future.'
+echo ' '
+echo 'The script assumes you have:'
+echo ' - installed the Google Cloud SDK: https://cloud.google.com/sdk/downloads'
+echo ' - subscribed your @gmail.com or other google account to M-Lab Discuss: '
+echo '   https://groups.google.com/a/measurementlab.net/forum/#!forum/discuss'
+echo ' - created a project for your Piecewise install in the Google Cloud Console: '
+echo '   https://console.cloud.google.com/'
+echo ' - configured your installed Google Cloud SDK with the Google account and '
+echo '   project name above'
+echo ' '
+echo 'If you need to cancel, press CTRL-C, complete the requirements above and '
+echo 'then re-run this script.'
+echo ' '
+read -p 'Name of your GCE VM instance: ' vmName
+read -p 'Image family (debian-8 or rhel-7): ' imageFam
+
+if [ $imageFam = 'debian-8' ]; then
+	imageProj='debian-cloud'
```

Comment:
Since we've decided to use Google style guidelines for pretty much everything, all indents should be 2 spaces.